server.js

```diff
@@ -124,7 +127,17 @@ app.post("/updateUserTag",async function(req, res){
     if (accessToken != '' && userAddress != '')
         await db.updateUserTag(userAddress, accessToken, tag)
     res.send({'status': 'success'});
-})
+});
+
+app.post('/formData', upload.single('imageURL'), async function (req, res) {
```

Comment:
change /formData to /createOrganization handle. /formData is too vague.

We'll need a change organization handle later on.